packages/react-components/src/components/styles/ChatMessageComponent.styles.ts

```diff
@@ -136,16 +134,19 @@ export const useChatMessageEditContainerStyles = makeStyles({
     ...shorthands.borderRadius(tokens.borderRadiusMedium),
     // The border thickness changes on hover, to prevent the border thickness change causing the
     // input box to shift we apply a margin to compensate. This margin is then removed on hover when the border is thicker.
-    ...shorthands.margin(`${defaultSendBoxActiveBorderThicknessREM - defaultSendBoxInactiveBorderThicknessREM}rem`),
+    marginBottom: '2px',
+    marginTop: '1rem',
     // Width should be updated on hover to include the border width change
     width: `calc(100% - ${defaultSendBoxActiveBorderThicknessREM}rem)`,
```

Comment:
This change breaks the current UI, the border makes the edit box to change position for a pixel or so on hover event. Steps to reproduce:
- open MessageThread preview in Storybook
- click on edit button for "your" message
- change focus from the message
- hover over the edit box (the blue border will shown) 
- hover over other messages
- check how it works on the current main branch